output/paper/TemplateTex.blg

```diff
@@ -0,0 +1,46 @@
+This is BibTeX, Version 0.99d (TeX Live 2021)
```

Comment:
Maybe gitignore the BLG file?

Also reminder to ignore the PDF files since we can't use git-lfs tracking in a Template.

Reply:
The blg file was already in the gitignore but had to be removed from the repo. Done in https://github.com/JMSLab/Template/pull/56/commits/1aafefda0f0c9a82a09ae0cffa06b458611081d6. I also removed the pdfs in https://github.com/JMSLab/Template/pull/56/commits/1aafefda0f0c9a82a09ae0cffa06b458611081d6 and added the pdfs in output/paper to the gitignore in https://github.com/JMSLab/Template/pull/56/commits/6b0f747a00d6a695a35108a22be5fa18ec354842.

Reply:
Actually, I changed how pdfs are ignored in https://github.com/JMSLab/Template/pull/56/commits/9af7f8d289b1be1c3ea5bbdb698cf368deb73936, so that rather than ignoring all .pdfs in output/paper, I just ignore the specific template pdfs by name. I also removed the other remaining pdfs in the repo in https://github.com/JMSLab/Template/pull/56/commits/9af7f8d289b1be1c3ea5bbdb698cf368deb73936.

@jmshapir fyi.